module/stats.py

```diff
@@ -1,15 +1,17 @@
 # Telegram
-from telegram import Update
+from telegram import Update, Bot
 from telegram.ext import CallbackContext
 
 # System libraries
 import sqlite3
+import os
 from datetime import date, timedelta
+import matplotlib.pyplot as plt
 
 def stats(update: Update, context: CallbackContext):
-    if(len(update['message']['text'].split(' ')) == 2):
-        days = int(update['message']['text'].split(' ')[1])
-        if(days <= 0):
+    if context.args:
+        days = int(context.args[0])
+        if days <= 0:
             days = 30
     else:
         days = 30
```

Comment:
```suggestion
    days = 30
    if context.args and int(context.args[0]) > 0:
        days = int(context.args[0])
```

che ne pensi? :grin: